Source/CarthageKit/Xcode.swift

```diff
@@ -151,6 +151,10 @@ private struct ProjectEnumerationMatch: Comparable {
 				return .success(ProjectEnumerationMatch(locator: .Workspace(URL), level: enumerator.level))
 			} else if (UTTypeConformsTo(typeIdentifier, "com.apple.xcode.project") != 0) {
 				return .success(ProjectEnumerationMatch(locator: .ProjectFile(URL), level: enumerator.level))
+			} else if (UTTypeConformsTo(typeIdentifier, "public.symlink") != 0) {
```

Comment:
Instead of explicitly handling symlinks here, could we just use `URL.URLByResolvingSymlinksInPath` before getting the resource value?